homeassistant/components/overkiz/number.py

```diff
@@ -37,9 +37,9 @@ class OverkizNumberDescription(NumberEntityDescription):
     min_value_state_name: str | None = None
     max_value_state_name: str | None = None
     inverted: bool = False
-    set_native_value: Callable[
-        [float, Callable[..., Awaitable[None]]], Awaitable[None]
-    ] | None = None
+    set_native_value: (
+        Callable[[float, Callable[..., Awaitable[None]]], Awaitable[None]] | None
+    ) = None
```

Comment:
Why is this change required?

Reply:
This is a work of ruff, not me :) Should I revert this block?

Reply:
Done